src/main.ts

```diff
@@ -30,7 +30,6 @@ interface App {
 }
 const ARCH = process.env.ARCH || 'linux';
 const githubToken = core.getInput('github-token');
-core.info(githubToken);
```

Comment:
Does it mean our githubToken has been logged somewhere ? Should rotate it?

Reply:
Looking at past executions I can't see it, it looks like github actions automatically scrubs it although I wasn't able to figure out how